src/components/progress-flow/progress-flow-item/progress-flow-item.tsx

```diff
@@ -49,26 +50,28 @@ export class ProgressFlowItem {
             return;
         }
 
-        return [
-            <button
-                tabindex="0"
-                title={this.getToolTipText()}
-                type="button"
-                class={{
-                    step: true,
-                    selected: this.item?.selected,
-                    disabled: this.isDisabled(),
-                    readonly: this.readonly,
-                }}
-                onClick={this.handleClick}
-                disabled={this.isDisabled()}
-            >
-                {this.renderIcon()}
-                <span class="text">{this.item.text}</span>
-                {this.renderDivider()}
-            </button>,
-            this.renderSecondaryText(),
-        ];
+        return (
+            <Host data-tracking-value={this.item?.value}>
```

Comment:
> So using data-tracking, and data-tracking-*, I think is a very acceptable way forward 😊

I've updated this.

Unless @jgroth or @vicgeralds wishes to veto this, I will approve an merge it.

Reply:
I would prefer if we did this in the parent component instead. Adding an attribute to `Host` is nearly as bad as having mutable props. Otherwise I think this ok. 🙂

Reply:
Yes I agree, and I think I mentioned that as well in an earlier discussion :smile:

Reply:
⚡

Reply:
I've approved it myself now 🙈, but please have a look!

Reply:
Ah, what the heck, it's a "private" change, so we can change or remove it without breaking anything other than the tracking, in which case we're not making anything worse than it already was without this. So I'm merging it now 😊